client/modules/core/main.js

```diff
@@ -487,6 +488,11 @@ export default {
     return Packages.findOne(query);
   },
 
+  getPackageSettingsWithOptions(options) {
+    const query = options;
```

Comment:
I did that just to make it clear what the options object was, but we don't really need it.